package.nls.json

```diff
@@ -80,7 +80,7 @@
     "AWS.configuration.description.awssam.debug.debuggingPlatform": "The debugger to be used with the Image-based Lambda Function",
     "AWS.configuration.description.awssam.debug.timeout": "The amount of time (in seconds) that Lambda allows a function to run before stopping it.",
     "AWS.configuration.description.awssam.debug.aws": "AWS connection details",
-    "AWS.configuration.description.awssam.debug.credentials": "The AWS credential type and ID to use during the invocation. Example: credential profile \"default\" would be entered as `profile:default`.",
+    "AWS.configuration.description.awssam.debug.credentials": "The AWS credential source and name to use during the invocation. Example: credential profile \"default\" would be entered as `staticProfile:default`.",
```

Comment:
This looks like a backwards incompatible change? What happens to people with `profile:foo` in their configurations when they run this code?

Reply:
Thanks for reviewing; we discussed this in https://github.com/aws/aws-toolkit-vscode/pull/1725#discussion_r626204148 , this PR is still WIP. 

I will preserve the `profile:` support. Though I'm thinking that there is no purpose in ever showing `profile:` to the user. That is the common case. The `source:` prefix is only needed for non-profile cases.

Reply:
I'm a bit confused on the use-case for the user needing to prepend the source type. Isn't every named profile being sourced from `credentials` and `config`? Is there a counterexample to this that's currently implemented?

Reply:
>  Is there a counterexample to this that's currently implemented?

We will implement "instance credentials" soon. In that case I guess the credential name will look like `ec2:default`.